proteus/mprans/RANS3PSed2D.h

```diff
@@ -1,12 +1,12 @@
-
 #ifndef RANS3PSed2D_H
 #define RANS3PSed2D_H
 #include <cmath>
 #include <iostream>
 #include "CompKernel.h"
 #include "ModelFactory.h"
 #include "SedClosure.h"
-
+#define DRAG_FAC 1.0
+#define TURB_FORCE_FAC 0.0
```

Comment:
Are these hacks?

Reply:
They are for debugging. The first turns off the physical drag and replaces  it with a simple global constant drag. The second turns off the turbulent force closure  relation.  I will turn the off before merging but probably leave them in the .h file for future testing.